dataservice/api/sequencing_center/resources.py

```diff
@@ -0,0 +1,166 @@
+from flask import abort, request
+from marshmallow import ValidationError
+from sqlalchemy.orm import Load
+
+from dataservice.extensions import db
+from dataservice.api.common.pagination import paginated, Pagination
+from dataservice.api.sequencing_center.models import SequencingCenter
+from dataservice.api.sequencing_center.schemas import (
+    SequencingCenterSchema
+)
+from dataservice.api.common.views import CRUDView
+
+
+class SequencingCenterListAPI(CRUDView):
+    """
+    SequencingCenter REST API
+    """
+    endpoint = 'sequencing_centers_list'
+    rule = '/sequencing-centers'
+    schemas = {'SequencingCenter': SequencingCenterSchema}
+
+    @paginated
+    def get(self, after, limit):
+        """
+        Get all sequencing_centers
+        ---
+        description: Get all sequencing_centers
+        template:
+          path:
+            get_list.yml
+          properties:
+            resource:
+              SequencingCenter
+        """
+        q = SequencingCenter.query
```

Comment:
Do the following so that you're loading the children kf_id's upfront and everything is in one query:
```python
SequencingCenter.query
             .options(joinedload(SequencingCenter.biospecimens).load_only('kf_id'))
```